internal/backend/sessions/sessionworkflows/workflows.go

```diff
@@ -167,6 +167,8 @@ func (ws *workflows) sessionWorkflow(wctx workflow.Context, params *sessionWorkf
 		ScheduleToCloseTimeout: ws.cfg.Temporal.LocalScheduleToCloseTimeout,
 	})
 
+	ctx := context.Background()
```

Comment:
why new context ?

Reply:
explain please?

Reply:
my bad, it was collapsed and I didn't see it's a new function

resolving